src/net/__net_init.cpp

```diff
@@ -0,0 +1,29 @@
+#include <net/__net_init.hpp>
+#ifdef _WIN32
+#include <WinSock2.h>
+#elif defined unix
```

Comment:
What's the point of these elifs? If you do nothing for linux, can't you just close if with endif, end of story?